pkgs/applications/audio/CHOWTapeModel/default.nix

```diff
@@ -0,0 +1,68 @@
+{ alsaLib
+, curl
+, fetchFromGitHub
+, freeglut
+, freetype
+, libGL
+, libXcursor
+, libXext
+, libXinerama
+, libXrandr
+, libjack2
+, pkgconfig
+, python3
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "CHOWTapeModel";
+  version = "unstable-2020-12-12";
+
+  src = fetchFromGitHub {
+    owner = "jatinchowdhury18";
+    repo = "AnalogTapeModel";
+    rev = "a7cf10c3f790d306ce5743bb731e4bc2c1230d70";
+    sha256 = "09nq8x2dwabncbp039dqm1brzcz55zg9kpxd4p5348xlaz5m4661";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+  buildInputs = [
+    alsaLib
+    curl
+    freeglut
+    freetype
+    libGL
+    libXcursor
+    libXext
+    libXinerama
+    libXrandr
+    libjack2
+    python3
+  ];
+
+  buildPhase = ''
+    cd Plugin/
+    ./build_linux.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib/lv2 $out/lib/vst3 $out/bin $out/share/doc/${pname}/
+    cd Builds/LinuxMakefile/build/
+    cp ${pname}.a  $out/lib
+    cp -r ${pname}.lv2 $out/lib/lv2
+    cp -r ${pname}.vst3 $out/lib/vst3
+    cp ${pname}  $out/bin
+    cp ../../../../Manual/ChowTapeManual.pdf $out/share/doc/${pname}/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/jatinchowdhury18/AnalogTapeModel";
+    description = "Physical modelling signal processing for analog tape recording. LV2, VST3 and standalone";
+    license = with licenses; [ gpl3 ];
```

Comment:
Plus or Only?

Reply:
Thanks for your tireless work @SuperSandro2000!

The other suggestions are now done.
I'll push a fix for this one when upstream responds.

Related, do you have an idea what is going on in [this issue](https://github.com/jatinchowdhury18/AnalogTapeModel/issues/164)?

Reply:
It's probably gpl3Only.

Reply:
OK, I pushed that.

Reply:
Turns out we guessed wrong...